coop_default_pricetag/data/product_print_category.xml

```diff
@@ -14,4 +14,38 @@
         <field name="field_ids" eval="[(4, ref('product.field_product_product__name')),(4, ref('product.field_product_product__list_price'))]" />
     </record>
 
+    <record id="product_product_pencl4_product_template" model="product.template">
+        <field name="name">Pen</field>
+        <field name="categ_id" ref="product.product_category_5"/>
+        <field name="standard_price">500.0</field>
+        <field name="list_price">750.0</field>
+        <field name="type">consu</field>
+        <field name="uom_id" ref="uom.product_uom_unit"/>
+        <field name="uom_po_id" ref="uom.product_uom_unit"/>
+        <field name="description_sale">160x80cm, with large legs.</field>
+        <field name="print_category_id" ref="ppc_demo_category"/>
+    </record>
+    <record id="product_product_pencl4" model="product.product">
+        <field name="name">Pen</field>
+        <field name="categ_id" ref="product.product_category_5"/>
+        <field name="list_price">500.0</field>
+        <field name="standard_price">750.0</field>
+        <field name="type">consu</field>
+        <field name="uom_id" ref="uom.product_uom_unit"/>
+        <field name="uom_po_id" ref="uom.product_uom_unit"/>
+        <field name="default_code">FURN_7800</field>
+        <field name="product_tmpl_id" ref="coop_default_pricetag.product_product_pencl4_product_template"/>
+    </record>
+    <record id="product_product_bookpcl4" model="product.product">
+        <field name="name">Book</field>
+        <field name="categ_id" ref="product.product_category_5"/>
+        <field name="list_price">450.0</field>
+        <field name="standard_price">300.0</field>
+        <field name="type">consu</field>
+        <field name="uom_id" ref="uom.product_uom_unit"/>
+        <field name="uom_po_id" ref="uom.product_uom_unit"/>
+        <field name="to_print">True</field>
+        <field name="print_category_id" ref="ppc_demo_category"/>
+    </record>
```

Comment:
This is data, not demo!
Please don't create products on data.

Do it on demo, if you need them for tests

Reply:
In the demo it's already there, I will remove products from the data file and improve the test case